tests/Model/LogEntryTest.php

```diff
@@ -18,4 +21,124 @@ public function testDoNotAllowEditing()
     {
         $this->assertFalse(LogEntry::create()->canEdit());
     }
+
+    /**
+     * Test the gridfield Summary method
+     */
+    public function testGridfieldSummary()
+    {
+        $shortString = 'abcdefghijklmnopqrstuvwxyz';
+        $longString = '';
+        // Generate a string of 350 characters. Because we don't care about the actual content
+        // A set of a's is good enough;
+        for ($i = 0; $i <= 350; $i++) {
+            $longString .= 'a';
+        }
+        $logEntry = LogEntry::create();
+        $logEntry->Entry = $shortString;
+        $this->assertEquals($shortString, $logEntry->getGridfieldSummary());
+        $logEntry->Entry = $longString;
+        $elipsisString = substr($longString, 0, 300) . '...';
+        $this->assertEquals($elipsisString, $logEntry->getGridfieldSummary());
+        $this->assertEquals($longString, $logEntry->getGridfieldSummary(351));
+    }
+
+    public function testEntryToUl()
+    {
+        $entry = array(
+            'Message'  => 'Something went wrong',
+            'DateTime' => '2016-01-22 03:14:23'
+        );
+        $logEntry = LogEntry::create(array(
+            'Level' => 'INFO',
+            'Entry' => Convert::array2json($entry)
+        ));
+        $result = $this->invokeMethod($logEntry, 'entryToUl', array($entry));
```

Comment:
I've been testing private/protected this way pretty much since I can remember starting with testing.

Although admitted, the invoker is something I pretty much google every time to copy-paste again from StackOverflow :P